apheleia-formatters.el

```diff
@@ -120,6 +120,8 @@
                 "--stderr" "--format" "quiet" "--fail-level" "fatal"))
     (ruby-standard . ("standardrb" "--stdin" filepath "--fix" "--stderr"
                       "--format" "quiet" "--fail-level" "fatal"))
+    (ruby-syntax-tree . ("stree" "format" filepath
+                         (apheleia-formatters-args-from-file ".streerc")))
```

Comment:
Is this based on some kind of standard? I would rather avoid Apheleia introducing new configuration files of its own, that won't be compatible with other tooling.

Reply:
The configuration file is optional and implemented based on the expectations described by [the Syntax Tree docs](https://github.com/ruby-syntax-tree/syntax_tree#configuration). I understand your concern though. Let me explore the possibility of removing the argument extraction from apheleia and instead provide a path to the rc file.

Reply:
Ah, I see, it is indeed a standard. We should not parse it ourselves, but we should make sure we run from the right working directory. Maybe the same fix as https://github.com/radian-software/apheleia/pull/239?